File: 1DAM/1DAM-Programacion/Unidad-8/FicherosJSONEmpleados/src/main/java/models/Departamento.java
package models;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;
import java.util.UUID;
//Usamos notaciones Lombock para disminuir la cantidad de código
@Data
@AllArgsConstructor
public class Departamento {
    private final String id = UUID.randomUUID().toString();
    private String Departamento;
    private List<Personas>personas;


}

File: 2DAM/AccesoADatos/Practicas/EmpleadoDepartamento/src/main/kotlin/models/Departamento.kt
package models

import java.util.UUID

data class Departamento(
    val id: UUID = UUID.randomUUID(),
    val nombre: String,
    val presupuesto: Double,
) {
    override fun toString(): String {
        return "Departamento(id=$id, nombre='$nombre', presupuesto=$presupuesto)"
    }
}